### ISSUE TITLE
Board page accessibility issues

### ISSUE THREAD

**Issue by gixxerblade** [Vets-Who-Code/vets-who-code-app#219]
# Accessibility 

#### Failing Elements

- Leads a research and engineering team as part of the Google Open Source Program…
`<p>Leads a research and engineering team as part of the Google Open Source Programs Office. </p>`
- Securing mission critical data wherever it resides through access controls, enc…
`<p>Securing mission critical data wherever it resides through access controls, encryption and key management.</p>`
- Experienced Business Executive and Sales Leader with a demonstrated history of …
`<p>Experienced Business Executive and Sales Leader with a demonstrated history of developing successful businesses and solutions across multiple industries. Currently serving as CEO of CoFlyt.</p>`
- Ela has an MBA and a degree in Psychology and over 20 years of experience worki…
`<p>Ela has an MBA and a degree in Psychology and over 20 years of experience working with aging individuals and their families and teaches at the University of Memphis</p>`
- Founder of Postlight with co-founder Rich Ziade. Postlight is middle-market fir…
`<p>Founder of Postlight with co-founder Rich Ziade. Postlight is middle-market firm focused on digital strategy. Defines, designs, and delivers digital platforms for clients like Vice News, the Obama Foundation, the NYC MTA, the Audubon Society, and dozens of others.</p>`
- Resourceful strategist with executive experience in project management, diversi…
`<p>Resourceful strategist with executive experience in project management, diversity &amp; sustainability initiatives, civics&nbsp;&amp; urban planning, and nonprofit operations &amp; governance.</p>`
- Passionate about the web, its reach, and how to create amazing experiences deli…
`<p>Passionate about the web, its reach, and how to create amazing experiences delivered in sub-second speeds. Worked on the highest trafficked websites in the world with Microsoft, Netflix, LinkedIn, and Reddit.</p>`
- A dedicated engineer consistently recruited for the toughest projects. Proven e…
`<p>A dedicated engineer consistently recruited for the toughest projects. Proven experience leading planning, project development, construction, operations, logistics, and public relations worldwide.</p>`
- Boston-based product designer with a background in research and front-end web d…
`<p>Boston-based product designer with a background in research and front-end web development.</p>`
- Christina Morillo is an information security, enterprise identity professional,…
`<p>Christina Morillo is an information security, enterprise identity professional, and technical product manager. She has worked for companies like Morgan Stanley, Fitch Ratings, AllianceBernstein, and currently Microsoft.</p>`
- Rachel is passionate about giving back to my community and her country. Much of…
`<p>Rachel is passionate about giving back to my community and her country. Much of her free time is dedicated to mentoring and helping others.</p>`
- Adept talent acquisition professional with 14+ years of experience with a passi…
`<p>Adept talent acquisition professional with 14+ years of experience with a passion for moving people from point A to B in their careers. Currently serving as a Technical Sourcer focused on the recruitment of Senior Engineers at Facebook.</p>`
- Author of Learn Python The Hard Way, Learn Python 3 The Hard Way, Learn C The H…
`<p>Author of Learn Python The Hard Way, Learn Python 3 The Hard Way, Learn C The Hard Way, Learn Ruby The Hard Way, and soon to be published, Learn More Python The Hard Way.</p>`
- Kyle is also well-known in the industry for teaching people web development thr…
`<p>Kyle is also well-known in the industry for teaching people web development through blogs, conference talks, meetups, and livestreaming on Twitch. </p>`
- Maxim Thorne is a philanthropist, lawyer, motivational speaker and consultant i…<p>
- Software Developer with over a decade of experience in creating web application…<p>
- Software Engineering Manager with a heavy background in front end development a…
`<p>Software Engineering Manager with a heavy background in front end development and a passion for bringing teams together.</p>`

#### Failing Elements

Amanda Casari
`<h5>Amanda<!-- --> <!-- -->Casari</h5>`

### Failing Elements
`<a rel="noopener noreferrer" href="https://www.linkedin.com/in/amcasari/" target="_blank">`
`<a rel="noopener noreferrer" href="https://twitter.com/amcasari" target="_blank">`
`<a rel="noopener noreferrer" href="https://www.linkedin.com/in/chris-christi-394a191/" target="_blank">`
`<a rel="noopener noreferrer" href="https://twitter.com/ChrisChristi1" target="_blank">`
`<a rel="noopener noreferrer" href="https://www.linkedin.com/in/tal-clark-a60776b/" target="_blank">`
`<a rel="noopener noreferrer" href="https://twitter.com/tal_clark?" target="_blank">`
`<a rel="noopener noreferrer" href="https://www.linkedin.com/in/elaemami/" target="_blank">`
`<a rel="noopener noreferrer" href="https://twitter.com/care2manage" target="_blank">`
`<a rel="noopener noreferrer" href="https://www.linkedin.com/in/ftrain/" target="_blank">`
`<a rel="noopener noreferrer" href="https://twitter.com/ftrain" target="_blank">`
`<a rel="noopener noreferrer" href="https://www.linkedin.com/in/leandregibson" target="_blank">`
`<a rel="noopener noreferrer" href="https://www.linkedin.com/in/btholt/" target="_blank">`
`<a rel="noopener noreferrer" href="https://twitter.com/holtbt" target="_blank">`
`<a rel="noopener noreferrer" href="https://www.linkedin.com/in/jon-korneliussen-p-e-a2b99167/" target="_blank">`
`<a rel="noopener noreferrer" href="https://twitter.com/JonKorneliussen" target="_blank">`
`<a rel="noopener noreferrer" href="https://www.linkedin.com/in/annalee921/" target="_blank">`
`<a rel="noopener noreferrer" href="https://www.linkedin.com/in/christinamorillo/" target="_blank">`
`<a rel="noopener noreferrer" href="https://twitter.com/divinetechygirl" target="_blank">`
`<a rel="noopener noreferrer" href="https://www.linkedin.com/in/rachelpolish/" target="_blank">`
`<a rel="noopener noreferrer" href="https://twitter.com/rachelpolish" target="_blank">`
`<a rel="noopener noreferrer" href="https://www.linkedin.com/in/alexreyes27/" target="_blank">`
`<a rel="noopener noreferrer" href="https://twitter.com/LEXNFX1429" target="_blank">`
`<a rel="noopener noreferrer" href="https://www.linkedin.com/in/zedshaw/" target="_blank">`
`<a rel="noopener noreferrer" href="https://twitter.com/lzsthw" target="_blank">`
`<a rel="noopener noreferrer" href="https://www.linkedin.com/in/kyleshevlin/" target="_blank">`
`<a rel="noopener noreferrer" href="https://twitter.com/kyleshevlin" target="_blank">`
`<a rel="noopener noreferrer" href="https://www.linkedin.com/in/maximthorne/" target="_blank">`
`<a rel="noopener noreferrer" href="https://twitter.com/maximthorne?lang=en" target="_blank">`
`<a rel="noopener noreferrer" href="https://www.linkedin.com/in/kennywheeler/" target="_blank">`
`<a rel="noopener noreferrer" href="https://twitter.com/ken_wheeler" target="_blank">`
`<a rel="noopener noreferrer" href="https://www.linkedin.com/in/caree-youngman-a87339ab/" target="_blank">`
`<a rel="noopener noreferrer" href="https://twitter.com/careecodes" target="_blank">`

**To Reproduce**
1. Go to 'https://vetswhocode.io/board'
2. Click on 'Developer Tools (F12)'
3. Click on Lighthouse
4. Generate report

**Expected behavior**
- Low contrast ratio - Text that has a low contrast ratio can be hard to read. Current contrast ratio is 3.78:1. A contrast ratio of at least 4.5:1 is recommended. 
- Heading elements - The element should be ordered correctly based on the current markup.
- Discernible names in anchor tags - Add `aria-label` to each `<a>` tag to make anchors more discernible.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
